Translations/hr.po

```diff
@@ -2,22 +2,22 @@
 # Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
 # This file is distributed under the same license as the PACKAGE package.
 # FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+# gogo <trebelnik2@gmail.com>, 2021.
 #
 msgid ""
 msgstr ""
 "Project-Id-Version: DevilutionX\n"
 "POT-Creation-Date: 2021-09-20 04:34+0200\n"
-"PO-Revision-Date: 2021-09-26 11:59+0200\n"
+"PO-Revision-Date: 2021-09-26 12:02+0200\n"
 "Last-Translator: gogo <trebelnik2@gmail.com>\n"
-"Language-Team: \n"
+"Language-Team: Croatian <hr@li.org>\n"
 "Language: hr\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"X-Generator: Poedit 2.3\n"
+"X-Generator: Gtranslator 40.0\n"
 "Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
-"%10<=4 && (n%100<12 || n%100>14) ? 1 : 2);\n"
-"X-Poedit-SourceCharset: UTF-8\n"
+"%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2)\n"
```

Comment:
Devilution is not able to parse the Plural-Forms rule and instead it simply string matches and uses a matching build in formula so we need to make sure this one is available.

Note to self: Produce a log warning if rule is not matched.